source/delegate/contracts/Delegate.sol

```diff
@@ -242,7 +242,7 @@ contract Delegate is IDelegate, Ownable {
       "AMOUNT_EXCEEDS_MAX");
 
     // Ensure the order is priced according to the rule.
-    require(order.sender.param == order.signer.param
+    require(order.sender.param <= order.signer.param
       .mul(10 ** rule.priceExp).div(rule.priceCoef),
       "PRICE_INCORRECT");
```

Comment:
Given that it's at-or-better than the quoted price, might now throw `PRICE_INVALID`

Reply:
As in rename the revert message?